lib/driver.js

```diff
@@ -634,6 +634,11 @@ class XCUITestDriver extends BaseDriver {
     log.info(`Simulator with udid '${this.opts.udid}' not booted. Booting up now`);
     await killAllSimulators();
     await this.opts.device.run(undefined, this.opts.allowTouchIdEnroll);
+
+    // Enable toggleEnrollTouchId initially
+    if (this.opts.allowTouchIdEnroll) {
+      await commands.toggleEnrollTouchId.call(this);
```

Comment:
Why the funkiness here? All the commands ought to have be put onto the driver object, so `this.toggleEnrollTouchId()` should suffice.

Also: If it toggles the enrollment, what happens if it is already enrolled. Won't this turn it off?